apps/web/src/common/modules/widgets/types/widget-field-value-type.ts

```diff
@@ -142,9 +143,14 @@ export interface LegendValue {
     position?: 'right' | 'bottom';
 }
 
+export type WidgetHeightValue = keyof typeof WIDGET_HEIGHT; // for Number Card Widget
+export interface WidgetHeight {
+    value: WidgetHeightValue;
+}
```

Comment:
suggestion: How about...
```typescript
export type WidgetHeightType = keyof typeof WIDGET_HEIGHT; // for Number Card Widget
export interface WidgetHeightValue {
    value: WidgetHeightType;
}
```

for convention😅

Reply:
OMG...